pyroma/tests.py

```diff
@@ -139,7 +140,8 @@ def test_minimal(self):
                     "Your package does not have keywords data.",
                     "Your package does not have author data.",
                     "Your package does not have author_email data.",
-                    "Your package should have a 'url' field with a link to the project home page, or a 'project_urls' field, with a dictionary of links, or both.",
+                    "Your package should have a 'url' field with a link to the project home page, or a "
```

Comment:
Just to note, it might be worth updating now before or when dropping the legacy processing (and updating the actual backend code accordingly), but the user-facing messages in `ratings.py` really should avoid referring to metadata fields using to to-be-deprecated, Setuptools-specific parameter names, as opposed to either the standardized source metadata key names in PEP 621 (probably best long-term, since that's what users are or will shortly be seeing in all modern tools) or the standard names of the core metadata fields.

For reference, PEP 621 (and PDM, Hatch and the other tools that use it), Flit and Poetry all have a `urls` _key_ to map to the `Project-URLs` _field_, equivalent to the Setuptools `project_url` _parameter_ (with no explicit mapping to the legacy `Home-page` field that the Setuptools `setup.cfg`/`setup.py` `url` parameter corresponds to), so this is likely to be confusing to users of any other tool, and even Setuptools itself once users adopt the recently-implemented PEP 621 support and `setup.cfg` is eventually deprecated, 

Also, perhaps a bit pedantic, but I'd really suggest using "field" correctly and consistently. "Field" refers to the core metadata _fields_, where the fields are called `Home-page` and `Project-URLs`, and nor are these the PEP 621 _key_, which is just called `urls`. Instead, these are the Setuptools _parameters_ `url` and `project_urls`.

Reply:
Ah, that's a good point. I'll have to check to include all supported name versions of metedata names. 

And yes, that is more pendantic that I will bother to try to be, because I will just mess that up anyway. :-D

Reply:
[PEP 621](https://peps.python.org/pep-0621/) gives a handy overview of the major build tools, as well as the PEP 621-standard name (which increasingly is becoming the only one that matters, especially with Setuptools' currently ongoing adoption of PEP 621 metadata and away from legacy `setup.cfg`/`setup.py`. But we're not quite there yet, unfortunately...

And fair enough; even the authors of PEP 621 mixed this up in some places, at least one of them causing major grief for me in PEP 639, because the difference was critical to the meaning of the interpretation (whether the `dynamic` key specified PEP 621 _keys_ or core metadata _fields_)